state_machine/applications/flight/Tasks/hw_monitor.py

```diff
@@ -0,0 +1,17 @@
+from Tasks.log import LogTask as Task
+from pycubed import cubesat
+from lib.alerts import alerts
+
+class task(Task):
+    name = 'hw_monitor'
+    color = 'orange'
+    data_file = None
+
+    async def main_task(self):
+        """
+        Prints the IMU data to the console.
```

Comment:
This comment doesn't seem to match the function

Reply:
thanks